code/game/machinery/vending/vendor_types/crew/staff_officer.dm

```diff
@@ -1,5 +1,53 @@
+//Staff officer Vendor//
+GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
+
+		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Service Uniform", 0, /obj/item/clothing/under/marine/officer/bridge, MARINE_CAN_BUY_UNIFORM, VENDOR_ITEM_MANDATORY),
+		list("Headset", 0, /obj/item/device/radio/headset/almayer/mcom, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
+		list("Satchel", 0, /obj/item/storage/backpack/satchel, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
```

Comment:
alphabetical order

Reply:
Question: As this was copy pasted from the XO vendor isn't it in this order for a reason? Much like how other vendors have Uni and boots before armor?

Reply:
I can also go fix all this for the XO vendor list if this has to do with some convention since most of it was copied from there

Reply:
Ah mb standard equipment doesn't need to be in alphabetical order (it's expected to be in gear up order)